# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/glmark2/hotfix-gcc12.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

From d1ca3f53c96dc8a4048b17dc16147a8fac782d4a Mon Sep 17 00:00:00 2001
From: Alexandros Frantzis <alexandros.frantzis@collabora.com>
Date: Mon, 14 Feb 2022 13:54:09 +0200
Subject: [PATCH] libmatrix: Add missing <utility> include

Fixes compilation with GCC 12.

Fixes #169
---
 src/libmatrix/program.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/libmatrix/program.h b/src/libmatrix/program.h
index f95b470c..1c9de08b 100644
--- a/src/libmatrix/program.h
+++ b/src/libmatrix/program.h
@@ -15,6 +15,7 @@
 #include <string>
 #include <vector>
 #include <map>
+#include <utility>
 #include "mat.h"
 
 // Simple shader container.  Abstracts all of the OpenGL bits, but leaves
